### DISCUSSION TITLE
how to get kubectl version

### DISCUSSION THREAD

**Discussion by mikethecalamity** [renovatebot/renovate#29710] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

I want to add a custom match string for the `kubectl` version, but I'm not sure the best datasource to use for it. It doesn't seem to be hosted on Github. They do have their latest stable version here: https://dl.k8s.io/release/stable.txt. But I'm not sure how to grab it from there.

```sh
# renovate: datasource=XXX ...
KUBECTL_VERSION=v1.28.11
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment, marked as answer]
you can use the kubernetes repo releases, they release kubectl together 

https://github.com/visualon/docker-buildpack/blob/4143cbfad9b02eff072b5ea09fe82bf3c1d20b2a/linux/test/run.sh#L15

**Reply by mikethecalamity** [reply]
Perfect thanks
